### PR TITLE
Rahel user test2 is done

### PR DESCRIPTION
**Pull request by Rahel-dev**
Its seems its working
test2 is done

### CODE REVIEW

**Comment by khomidov-byte** [comment]
test 2 it's just started

File: src/rahel/Test2.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package rahel;
+
+public class Test2 {
+    //its done
```
Comment (khomidov-byte) [inline src/rahel/Test2.java:4]: // just done

**Review by khomidov-byte — COMMENTED** [review]
Cool code

**Review by BojanTrboljevac — APPROVED** [review]
great work

**Review by Dancibela87 — APPROVED** [review]
approved

**Pull request merged**